### ISSUE TITLE
homebridge-hue@0.11.54 requires homebridge@0.4.51 but latest version appears to be 0.4.50

### ISSUE THREAD

**Issue by thumax** [ebaauw/homebridge-hue#624]
#### Issue

While homebridge@0.4.51 is required, the latest version available at npm appears to be 0.4.50. 

Sorry if I'm that blind guy who can't find a current homebridge release, but maybe this issue can help the hordes of similar simpletons who will follow without doubt. ;-)
#### Log Messages

```
Error: Plugin /usr/lib/node_modules/homebridge-hue requires a HomeBridge version of ^0.4.51 which does not satisfy the current HomeBridge version of 0.4.50. You may need to upgrade your installation of HomeBridge.
    at Plugin.load (/usr/lib/node_modules/homebridge/lib/plugin.js:56:11)
    at Server.<anonymous> (/usr/lib/node_modules/homebridge/lib/server.js:153:14)
    at Array.forEach (<anonymous>)
    at Server._loadPlugins (/usr/lib/node_modules/homebridge/lib/server.js:145:22)
    at new Server (/usr/lib/node_modules/homebridge/lib/server.js:57:24)
    at module.exports (/usr/lib/node_modules/homebridge/lib/cli.js:32:16)
    at Object.<anonymous> (/usr/lib/node_modules/homebridge/bin/homebridge:17:22)
    at Module._compile (internal/modules/cjs/loader.js:1158:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1178:10)
    at Module.load (internal/modules/cjs/loader.js:1002:32)
    at Function.Module._load (internal/modules/cjs/loader.js:901:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:74:12)
    at internal/main/run_main_module.js:18:47
```

**Comment by beatpaul** [comment]
Same for me, this version crashes my home bridge hue instance :(
```
/Users/earth/.pm2/logs/INS0-error.log last 15 lines:
0|INS0     |     at Object.Module._extensions..js (internal/modules/cjs/loader.js:991:10)
0|INS0     |     at Module.load (internal/modules/cjs/loader.js:811:32)
0|INS0     | [2020-2-20 22:33:53] ====================
0|INS0     | [2020-2-20 22:33:53] Error: The requested platform 'Hue' was not registered by any plugin.
0|INS0     |     at API.platform (/usr/local/lib/node_modules/homebridge/lib/api.js:134:13)
0|INS0     |     at Server._loadPlatforms (/usr/local/lib/node_modules/homebridge/lib/server.js:327:45)
0|INS0     |     at Server.run (/usr/local/lib/node_modules/homebridge/lib/server.js:90:36)
0|INS0     |     at module.exports (/usr/local/lib/node_modules/homebridge/lib/cli.js:59:10)
0|INS0     |     at Object.<anonymous> (/usr/local/lib/node_modules/homebridge/bin/homebridge:17:22)
0|INS0     |     at Module._compile (internal/modules/cjs/loader.js:955:30)
0|INS0     |     at Object.Module._extensions..js (internal/modules/cjs/loader.js:991:10)
0|INS0     |     at Module.load (internal/modules/cjs/loader.js:811:32)
0|INS0     |     at Function.Module._load (internal/modules/cjs/loader.js:723:14)
0|INS0     |     at Function.Module.runMain (internal/modules/cjs/loader.js:1043:10)
0|INS0     | [2020-2-20 22:33:54] [Config_0] Homebridge Config UI X v4.10.3 is listening on :: port 8080


```
Running it on a mac with several instances

**Comment by thumax** [comment]
I changed the dependency version back to 0.4.50 in my local package.json and homekit-hue appears to be running for now. At least homebridge starts up without complaining and I can control my non-hue ZigBee lights again. ;-)

Stil would be interested in how to properly solve this issue.

**Comment by ebaauw** [comment]
I must have been looking cross-eyed, and replaced the wrong `0` with a `1`.

Fixed in v0.11.55 and unpublished v0.11.54.  Thanks for the quick catch.

**Comment by thumax** [comment]
Perfect, thank you! :-)
